website/docs/components/super-select/partials/code/component-api.md

```diff
@@ -0,0 +1,38 @@
+## Component API
+
+SuperSelect consists of two sub-components which are used directly.
+
+See the Ember [PowerSelect documentation](https://ember-power-select.com/docs) for the full list of supported options.
+
+### SuperSelect::Single
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="renderInPlace" @type="boolean" @default="true">
+    Determines whether the dropdown options are rendered in place within the DOM vs. being attached to the root of the HTML `<body>` and positioned with JavaScript.
+    <br><br>
+    Defined as `true` by default in `SuperSelect` components to fix a `PowerSelect` accessibility issue. (The opener element or “trigger” should be next to dropdown content it hides & reveals.)
```

Comment:
@MelSumner Could you please take a look at how I worded this accessibility issue here? I think it could be improved. Thanks!

(I repeated the same text below for the `SuperSelect::Multiple` API.)

Reply:
Resolving as outdated